releng/org.eclipse.xtext.sdk.parent/pom.xml

```diff
@@ -28,7 +28,7 @@
 			</plugin>
 			<plugin>
 				<groupId>org.eclipse.tycho</groupId>
-				<artifactId>tycho-versions-plugin</artifactId>
```

Comment:
is this change intentional?

Reply:
To be honest, not absolutly sure. Done this together with @kthoms and have not a full understanding of this detail.